packages/cli/src/commands/generate/page/page.js

```diff
@@ -197,5 +197,6 @@ export const handler = async ({
     await tasks.run()
   } catch (e) {
     console.log(c.error(e.message))
```

Comment:
Use stderr, instead of stdout.
```suggestion
    console.error(c.error(e.message))
```